Doc/library/mailcap.rst

```diff
@@ -6,6 +6,10 @@
 
```

Comment:
We added this to the other modules:
```suggestion
   :deprecated:

```

Reason was to make it say "deprecated" in the module index: https://github.com/python/cpython/pull/31891#issuecomment-1068354411

But because we're adding these to superseded.rst looks like it's not shown at `/library/index.html`. Anyway, we might as well include it in case it's used somewhere else!

Reply:
Ok, added.